### PR TITLE
[FIX] payment: issue Authorize.net manual capture

### PR DESCRIPTION
**Pull request by hote-odoo**
When a 'manual capture' is set for Authorize.net and a client
make a payment in several partial payments the user can capture
all the authorized transactions without crashing now.

Task - 2676914

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/payment/models/account_invoice.py [hunk 1]
```diff
@@ -88,7 +88,8 @@ def _create_payment_transaction(self, vals):
         return transaction
 
     def payment_action_capture(self):
-        self.authorized_transaction_ids.s2s_capture_transaction()
+        for transaction in self.authorized_transaction_ids:
+            transaction.action_capture()
```
Comment (Feyensv) [inline addons/payment/models/account_invoice.py:92]:
```suggestion
        self.authorized_transaction_ids.action_capture()
```
No need to iterate on the transaction, `action_capture` supports being called with 0, 1 or n records :).

File: addons/sale/models/sale.py [hunk 2]
```diff
@@ -1095,7 +1095,8 @@ def _force_lines_to_invoice_policy_order(self):
                 line.qty_to_invoice = 0
 
     def payment_action_capture(self):
-        self.authorized_transaction_ids.s2s_capture_transaction()
+        for transaction in self.authorized_transaction_ids:
+            transaction.action_capture()
```
Comment (Feyensv) [inline addons/sale/models/sale.py:1099]: Same remark

**Review by Feyensv — COMMENTED** [review]
Can be simplified a little bit ;)

File: addons/payment/models/account_invoice.py (hunk 1 above)
Comment (hote-odoo) [reply addons/payment/models/account_invoice.py]: thanks !

File: addons/sale/models/sale.py (hunk 2 above)
Comment (hote-odoo) [reply addons/sale/models/sale.py]: thanks again :)

File: addons/payment/models/account_invoice.py [hunk 3]
```diff
@@ -88,7 +88,7 @@ def _create_payment_transaction(self, vals):
         return transaction
 
     def payment_action_capture(self):
-        self.authorized_transaction_ids.s2s_capture_transaction()
+        self.authorized_transaction_ids.action_capture()
 
     def payment_action_void(self):
         self.authorized_transaction_ids.s2s_void_transaction()
```
Comment (AntoineVDV) [inline addons/payment/models/account_invoice.py:94]: Shouldn't we apply the exact same fix for the void action?

File: addons/sale/models/sale.py [hunk 4]
```diff
@@ -1095,7 +1095,7 @@ def _force_lines_to_invoice_policy_order(self):
                 line.qty_to_invoice = 0
 
     def payment_action_capture(self):
-        self.authorized_transaction_ids.s2s_capture_transaction()
+        self.authorized_transaction_ids.action_capture()
 
     def payment_action_void(self):
         self.authorized_transaction_ids.s2s_void_transaction()
```
Comment (AntoineVDV) [inline addons/sale/models/sale.py:1101]: Same here.

**Review by AntoineVDV — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**